MembershipApp/templates/MembershipApp/about.html

```diff
@@ -0,0 +1,134 @@
+                        <div class="text-block">
+                            <h5>SPEAK AT BUBBLY EVER AFTER </h5>
+                            <p>
+                                <font color="#515151">Please </font><a
+                                    href="https://accounts.google.com/signin/v2/identifier?service=wise&amp;passive=1209600&amp;continue=https%3A%2F%2Fdocs.google.com%2Fspreadsheets%2Fd%2F188hmLwwL_h-4FlKfq675TmoP0vlZFebk-umBQ1FlzqY%2Fedit%3Fusp%3Dsharing&amp;followup=https%3A%2F%2Fdocs.google.com%2Fspreadsheets%2Fd%2F188hmLwwL_h-4FlKfq675TmoP0vlZFebk-umBQ1FlzqY%2Fedit%3Fusp%3Dsharing&amp;ltmpl=sheets&amp;flowName=GlifWebSignIn&amp;flowEntry=ServiceLogin">sign
+                                up here</a>
+                                <font color="#515151"> if you'd like to be considered to host a talk, workshop or
+                                    experience in line with our speaker series (which will be made public soon).</font>
+                            </p>
+                        </div>
+                        <div class="text-block">
+                            <h5>SHARE WITH BUBBLY EVER AFTER</h5>
+                            <p>
+                                <font color="#515151">Our main lounge will also be holding space for groups,
+                                    individuals, and musicians who want to host meetings, talks, workshops, and
+                                    impromptu jams when the space is not in use. Please let us know if you'd like to use
+                                    the space for facilitation.</font><br><br>
+                                <font color="#515151">We love cross-camp and good human collaboration. </font><br>
+                                <font color="#515151">If your camp would like to come on by with an art car, a Happy
+                                    Hour, a dance, art, music, etc to share with others. Please let us know. We'd love
+                                    to clear some space and have you come share some magic with us in our communal
+                                    space.</font>
+                            </p>
+                            <p><b>DONATE</b><br><br>
+                                <font color="#515151">Your gifts will fund the structures, speaker series, lounges, art,
+                                    and the bubbly on the playa and beyond. They will be much
+                                    appreciated.</font><br><br>
+                                <font color="#515151">Gifts can be made</font><br><br>
+                                <font color="#515151">via </font><a href="https://www.paypal.com/">PayPal</a>
+                                <font color="#515151"> to everbubblyafter@gmail.com.&nbsp;</font><br></p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+{% endblock %}
+<footer class="space--sm footer-2">
```

Comment:
please abstract footer to it's own file and include it in all templates. Like this changes to footer can be made in one place

Reply:
apply this to all templates